src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -346,8 +336,8 @@ class QueryAppsDataValidatorTemplate {
             }
             for (std::size_t idx = 0; idx < syn_size; ++idx) {
               const smart_objects::SmartObject& synonym = (*syn_array)[idx];
-              const utils::custom_string::CustomString vrSynonym =
-                  synonym.asCustomString();
+              std::string vrSynonym =
```

Comment:
@okoreniak The same as above. Can you make it `const`? Or you are changing its value afterwards?